app/controllers/api/v1x0/requests_controller.rb

```diff
@@ -32,7 +32,9 @@ def index_scope(relation)
       private
 
       def request_params
-        params.permit(:name, :description, :tag_resources => [], :content => {})
+        params.permit(:name, :description, :workflow_id, :requester_name,
```

Comment:
@hsong-rh Whereever request_params is being used we can use 
https://github.com/ManageIQ/manageiq-api-common/blob/8e76ff7192ed56d84a64000dab7c9f0dc80534cc/lib/manageiq/api/common/application_controller_mixins/parameters.rb#L11
Since your controllers are using that mixing
An example is here
https://github.com/ManageIQ/catalog-api/blob/d2a85e62b47b060631554ba29762a2f71875966a/app/controllers/api/v1/portfolio_items_controller.rb#L28

Reply:
👍 Will change accordingly after https://github.com/ManageIQ/manageiq-api-common/pull/133 is merged.